### ISSUE TITLE
Open: No such file or directory while running psiclass/classes

### ISSUE THREAD

**Issue by Atari61** [splicebox/PsiCLASS#22]
Hi, 
I'm using PsiClass on 12 STAR bams. Stages from 0 to 3 run without any problems, but 4th throws the following error:

```
/storage/home/maslov/psiclass/classes  -p 28  --lb /storage/analysis/opi/transcriptome/class/bamlist.txt  -s ./subexon/psiclass_subexon_combined.out -o ./psiclass > ./psiclass_classes.log
open: No such file or directory
Segmentation fault (core dumped)
Terminated
```

The command I use to run PsiClass:
`psiclass --lb $WD/bamlist.txt --bamGroup $WD/bamgroup.txt -p 28
`
This error replicates when I run the classes command alone.

On the other hand, example data works just fine and doesn't throw any error.

Thank you very much for your help

**Comment by mourisl** [comment]
Could you please check whether the ./subexon/psiclass_subexon_combined.out file exists? Is there any output information in the ./psiclass_classes.log file? Thank you.

**Comment by Atari61** [comment]
`./subexon/psiclass_subexon_combined.out` exists and is not empty, `./psiclass_classes.log` also exists but is empty.

**Comment by mourisl** [comment]
Could you please directly run "/storage/home/maslov/psiclass/classes  -p 28  --lb /storage/analysis/opi/transcriptome/class/bamlist.txt  -s ./subexon/psiclass_subexon_combined.out -o ./psiclass" at that folder and see what is the output on the screen?

**Comment by Atari61** [comment]
I ran it and got the same output:
```
/storage/analysis/opi/transcriptome/class$ /storage/home/maslov/psiclass/classes -p 28 --lb /storage/analysis/opi/transcriptome/class/bamlist.txt -s ./subexon/psiclass_subexon_combined.out -o ./psiclass
open: No such file or directory
Segmentation fault (core dumped)
```

**Comment by mourisl** [comment]
For the full psiclass run, did you find other such errors in the stage of "junc" and "subexon-info"? It could be that one of the bam files in the bamlist.txt is inaccessible.

**Comment by Atari61** [comment]
No, I haven't. Just a clean run, expect the `classes` part: for each of the bam files in bamlist.txt there's both .splice and .raw_splice and a subexon_${num}_out, at least.
I just have tried a set of hisat bams and got the same error and the same output.

**Comment by Atari61** [comment]
Okay, I've mved all the input to the installation dir and have run PsiClass from here and it classes works now.

**Comment by mourisl** [comment]
It should be run from anywhere. How long is each line in the bamlist.txt file? It could be the path to a bam file is too long (>1024 characters) and causes the error.

**Comment by Atari61** [comment]
Each line has 90-ish chars.
